README.md

```diff
@@ -27,7 +27,6 @@ All contributors to any Work Items must be members of the Solid CG. It’s easy
 Anyone can join the [Authorization Panel chat](https://gitter.im/solid/authorization-panel).
 
 Online panel meetings occur sporadically on Wednesday, 14:00 UTC at https://meet.jit.si/solid-authorization.
```

Comment:
```suggestion
Online panel meetings occur sporadically on Wednesday, 14:00 UTC at https://meet.jit.si/solid-authorization.

For now, a meeting may be requested by raising an issue on the 
[`authorization-panel` repo](https://github.com/solid/authorization-panel/issues/) 
or posting to the [Solid CG mailing list](mailto:public-solid@w3.org).

```

Reply:
I would rather keep issues to their current and primary use and I don't follow the Solid CG mailing list.

Links to HackMD and discussions about occurence of meetings have always taken place on Gitter until now, I don't see a very compelling reason to change the way things have worked until now. Especially for a temporary measure.

Reply:
I honestly doubt that anyone will request the meeting, especially since we want to resolve the stable schedule ASAP.

I suggest merging the PR as it is and if @TallTed feels strongly about this amendment it could be proposed in a separate PR.

Reply:
Sounds good to me @elf-pavlik .